<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EasyNotesClass</class>
 <widget class="QWidget" name="EasyNotesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyNotes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="EasyNotes.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>7</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="clearButton">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>clear</string>
        </property>
        <property name="icon">
         <iconset resource="EasyNotes.qrc">
          <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="saveButton">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>save</string>
        </property>
        <property name="icon">
         <iconset resource="EasyNotes.qrc">
          <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="pinButton">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="EasyNotes.qrc">
          <normaloff>:/images/固定,图钉.png</normaloff>
          <normalon>:/images/解除固定,图钉.png</normalon>:/images/固定,图钉.png</iconset>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="EasyNotes.qrc"/>
 </resources>
 <connections/>
</ui>
